nixnet/_enums.py

```diff
@@ -2029,3 +2029,26 @@ class FrameType(enum.Enum):
     SPECIAL_DELAY = _cconsts.NX_FRAME_TYPE_SPECIAL_DELAY
     SPECIAL_LOG_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_LOG_TRIGGER
     SPECIAL_START_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_START_TRIGGER
+
+class Terminals(enum.Enum):
+    "Terminals to import/export from the XNET device"
+    NX_TERM_FRONTPANEL0 = _cconsts.NX_TERM_FRONT_PANEL0
```

Comment:
1. Because these are in an enum, we shouldn't have the `NX_TERM` / `NXTERM` prefixes
2. Why did `FRONT_PANEL0` get turned into `FRONTPANEL0`?  And should that be `_0`?

Reply:
For the first pass I matched the names with their C counterparts for consistency. I don't have a strong argument for this approach and have no problem diverging to improve readability.